packages/quantic/force-app/main/default/lwc/quanticColoredResultBadge/quanticColoredResultBadge.js

```diff
@@ -30,22 +41,34 @@ export default class QuanticColoredResultBadge extends LightningElement {
   /** @type{boolean} */
   invalidColor = false;
 
+  hasValidTextContent() {
+    return !!this.label || !!this.result?.raw?.[this.field]
+  }
```

Comment:
This property duplicates `textToDisplay` except it returns a boolean instead of a string. I'd suggest to use similar names for both properties, and to reuse `textToDisplay`.

```suggestion
  hasTextToDisplay() {
    return !!this.textToDisplay;
  }
```

However, this is subject to changes when incorporating other components to format the field values.